syntax = "proto3";

service CommentAnalysis {
    rpc AnalyzeComment (CommentRequest) returns (CommentResponse);
}

message CommentRequest {
    int32 comment_id = 1;
    string text = 2;
}

message CommentResponse {
    int32 comment_id = 1;
    string topic_id = 2;
    string opinion_class = 3;
    string conclusion = 4;
}
